backend/uploader.py

```diff
@@ -22,19 +21,24 @@ def save_file_to_db(filename):
 
         if error == None:
             jsonfile.seek(0)
-        
+
             manifest = json5.load(jsonfile)
             jsonfile.close()
             manifest['date_creation'] = time.strftime("%Y-%m-%d")
             manifest['date_update'] = time.strftime("%Y-%m-%d")
             manifest['id'] = uuid.uuid4()
 
-            elastic.store_json('test', 'projects', manifest)
-            coll.insert_one(manifest)
-            print("Successfully validated file. ID is " + str(manifest['id']),file=sys.stderr)
-            print("File content is: ", file=sys.stderr)
-            print(manifest, file=sys.stderr)
-            return manifest['id']
+            res = es.index(index="projects-index", doc_type='Project', id=manifest['id'], body=manifest)
+            if res['created']:
+                coll.insert_one(manifest)
+
+                print("Successfully validated file. ID is " + str(manifest['id']),file=sys.stderr)
+                print("File content is: ", file=sys.stderr)
+                print(manifest, file=sys.stderr)
+                return manifest['id']
+            else:
+                print(error,file=sys.stderr)
+                raise ApiException("ElasticSearch Index Error: \n" + str(error), 500)
```

Comment:
res['created'] also returns false if the entry was updated, Printing the error variable inside an if error=None: block is useless, especially if it hasn't necessarily been an error. I did not think about the issue of updating projects via uploader, so thank you for bringing this to my attention, I will fix it next week.